python/katana/cpp/libgalois/Galois.pxd

```diff
@@ -1,8 +1,11 @@
 from ..libstd cimport CPPAuto
+from libcpp.string cimport string
 
 cdef extern from "katana/Galois.h" namespace "katana" nogil:
     unsigned int setActiveThreads(unsigned int)
 
+    string getVersion()
```

Comment:
Since I'm asking you to remove this function from `Galois.h`, move `getVersion` out of this block and into it's own `cdef extern` block that is `from "katana/Version.h"`.